### PR TITLE
cc36 done

### PR DESCRIPTION
**Pull request by ryanapodaca**

### CODE REVIEW

**Comment by ryanapodaca** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: java/datastructures/lib/src/main/java/codechallenges/graph/BreadthTraversal.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class BreadthTraversal {
+  public static ArrayList<String> getGraphValues(Graph graph) {
+    if(graph == null) {
+      return null;
+    }
+    Queue<Vertex<String>> unvisitedQueue = new Queue<>();
+    Set<Vertex<String>> visitedHashSet = new HashSet<>();
+    ArrayList<String> graphValues = new ArrayList<>();
+
+    LinkedList<Vertex<String>> vertexes = graph.getVertices();
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/main/java/codechallenges/graph/BreadthTraversal.java:19]: What's tripping you up here is the inherent lack of order in graphs & hash maps. To the human brain, you put Pandora in first, so it should come out first, but the order of the items is actually determined by their keys. Because the keys are randomly hashed, it's impossible to predict which vertex will be at the front of this list.

In this case, when I ran your test through the debugger, the first node in the `vertexes` list is Arendelle, so the traversal starts at Arendelle. Since the traversal looks at the *destination* of each edge, and Pandora is not the destination of any edge, Pandora never gets visited.

The way I approached this was to pass the starting vertex into `getGraphValues` instead of the whole graph. That way, you can control exactly which vertex the traversal starts at. 

You could also use your current method as-is and just change your test graph so that every vertex is the destination of at least one edge.
